packages/bootstrap/extra/dcos_internal_utils/cli.py

```diff
@@ -59,6 +59,8 @@ def noop(b, opts):
     'dcos-minuteman': noop,
     'dcos-navstar': noop,
     'dcos-spartan': noop,
+    'dcos-log': noop,
```

Comment:
nit: can we call this `dcos-log-master` for consistency between this, the unit names?